config/bidderinfo.go

```diff
@@ -201,6 +201,21 @@ func (bi BidderInfo) IsEnabled() bool {
 	return !bi.Disabled
 }
 
+// Defined returns true if at least one field exists, except for the supports field.
+func (s *Syncer) Defined() bool {
+	if s == nil {
+		return false
+	}
+
+	return s.Key != "" ||
```

Comment:
For the purposes of this check, do we care only that at least one field has a non-default value, or do we want to verify that there is enough to actually use? For example, we need one of IFrame and Redirect to be non-nil. We can't do much if everything except SupportCORS is nil.

Reply:
Hm good question. I think the presence of any of these fields implies that there is intent to define a user sync. If it is partially defined but not good enough to actually use I suppose we should create the user sync and expect the failure on startup. That's how you appear to have coded it @SyntaxNode. Is that your take?

Reply:
Correct. This is our best attempt to determine if the human writing the config for the bidder intends for the usersync to work.